### PR TITLE
Simplify `AxesActor` label property names

### PR DESCRIPTION
**Pull request by user27182**
### Overview

Maintenance as part of #5029. Renames `x_axis_label` -> `x_label`, for example.

The existing property names are a bit verbose and not consistent with other axes-related properties.

### CODE REVIEW

**Comment by user27182** [comment]
pre-commit.ci autofix

File: pyvista/plotting/axes_actor.py [hunk 1]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+
+    @y_axis_label.setter
+    def y_axis_label(self, label: str):  # numpydoc ignore=GL08
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `y_axis_label` is deprecated. Use `y_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):
+            raise RuntimeError('Remove this deprecated property')
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:508]:
```suggestion
        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
            raise RuntimeError('Remove this deprecated property')
```

File: pyvista/plotting/axes_actor.py [hunk 2]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+            This parameter is deprecated. Use :attr:`z_label` instead.
+
+        """
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `z_axis_label` is deprecated. Use `z_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):
+            raise RuntimeError('Remove this deprecated property')
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:545]:
```suggestion
        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
            raise RuntimeError('Remove this deprecated property')
```

File: pyvista/plotting/axes_actor.py [hunk 3]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+
+    @z_axis_label.setter
+    def z_axis_label(self, label: str):  # numpydoc ignore=GL08
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `z_axis_label` is deprecated. Use `z_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):
+            raise RuntimeError('Remove this deprecated property')
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:556]:
```suggestion
        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
            raise RuntimeError('Remove this deprecated property')
```

**Review by tkoyama010 — APPROVED** [review]
+1 to change name. LGTM.

File: pyvista/plotting/axes_actor.py [hunk 4]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+
+        """
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `x_axis_label` is deprecated. Use `x_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
+            raise RuntimeError('Remove this deprecated property')
+        return self.GetXAxisLabelText()
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:450]:
```suggestion
        return self.GetXAxisLabelText()  # pragma: no cover
```

File: pyvista/plotting/axes_actor.py [hunk 5]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+    @x_axis_label.setter
+    def x_axis_label(self, label: str):  # numpydoc ignore=GL08
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `x_axis_label` is deprecated. Use `x_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
+            raise RuntimeError('Remove this deprecated property')
+        self.SetXAxisLabelText(label)
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:461]:
```suggestion
        self.SetXAxisLabelText(label)  # pragma: no cover
```

File: pyvista/plotting/axes_actor.py [hunk 6]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+
+        """
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `y_axis_label` is deprecated. Use `y_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
+            raise RuntimeError('Remove this deprecated property')
+        return self.GetYAxisLabelText()
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:498]:
```suggestion
        return self.GetYAxisLabelText()  # pragma: no cover
```

File: pyvista/plotting/axes_actor.py [hunk 7]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+    @y_axis_label.setter
+    def y_axis_label(self, label: str):  # numpydoc ignore=GL08
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `y_axis_label` is deprecated. Use `y_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
+            raise RuntimeError('Remove this deprecated property')
+        self.SetYAxisLabelText(label)
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:509]:
```suggestion
        self.SetYAxisLabelText(label)  # pragma: no cover
```

File: pyvista/plotting/axes_actor.py [hunk 8]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+
+        """
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `z_axis_label` is deprecated. Use `z_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
+            raise RuntimeError('Remove this deprecated property')
+        return self.GetZAxisLabelText()
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:546]:
```suggestion
        return self.GetZAxisLabelText()  # pragma: no cover
```

File: pyvista/plotting/axes_actor.py [hunk 9]
```diff
@@ -417,73 +419,159 @@ def labels(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
...
+    @z_axis_label.setter
+    def z_axis_label(self, label: str):  # numpydoc ignore=GL08
+        # deprecated 0.44.0, convert to error in 0.46.0, remove 0.47.0
+        warnings.warn(
+            "Use of `z_axis_label` is deprecated. Use `z_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        if pyvista._version.version_info >= (0, 47):  # pragma: no cover
+            raise RuntimeError('Remove this deprecated property')
+        self.SetZAxisLabelText(label)
```
Comment (tkoyama010) [inline pyvista/plotting/axes_actor.py:557]:
```suggestion
        self.SetZAxisLabelText(label)  # pragma: no cover
```

**Review by tkoyama010 — APPROVED** [review]

**Pull request merged**